gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/RunTask.groovy

```diff
@@ -23,9 +23,10 @@ class RunTask {
 
     static JavaExec createRunTask(Project project, String taskName) {
         return project.tasks.create(taskName, JavaExec) { t ->
+            t.dependsOn(project.tasks.jar)
```

Comment:
Which behavior? I've tested this against one of the projects which uses this plugin and leaving out the `dependsOn(jar)` broke it